src/sdg/organisaties/models.py

```diff
@@ -110,19 +110,22 @@ class Lokatie(models.Model):
     straat = models.CharField(
         _("straat"),
         max_length=256,
+        help_text=_("De straatnaam van de lokatie."),
     )
     nummer = models.PositiveIntegerField(
         _("nummer"),
+        help_text=_("Het huisnummer van de lokatie."),
+    )
+    plaats = models.CharField(
+        _("plaats"),
+        max_length=256,
+        help_text=_("De plaatsnaam van de lokatie."),
```

Comment:
Mot sure why you moved plaats. The order was good here.

Dutch addresses are typically:

```
<straat> <nummer>
<postcode> <plaats>
```

Reply:
The reason for this change is due to the ordering on the form:

![location form](https://user-images.githubusercontent.com/58147939/147222016-af65a631-bea9-4f62-9231-04fa0a081ab0.png)